src/core/zowe/core_for_zowe_sdk/logger.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import os
+
+class Log:
+    """root logger setup and a funtion to customize logger level"""
+
+    dirname = os.path.join(os.path.expanduser("~"), ".zowe/logs")
+
+    if not os.path.isdir(dirname):
+        os.makedirs(dirname)
+
+    logging.basicConfig(
+        filename=os.path.join(dirname, "python_sdk_logs.log"),
+        level=logging.INFO,
+        format="[%(asctime)s] [%(levelname)s] [%(name)s] - %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S %p",
+    )
```

Comment:
If all the logs are written to the same file, over a long period of time it will grow in size and become many MB. We may want to investigate the `RotatingFileHandler` class which switches logging to a new file when the file exceeds a certain size, and deletes old files (similar to what our Node.js SDK does).

Since logging to a file is working, this could be addressed in a separate issue 🙂

Reply:
It is a good idea. Maybe a separate issue?